extension/src/popup/components/signTransaction/Operations/styles.scss

```diff
@@ -42,6 +42,7 @@
     font-size: 0.875rem;
     height: 7rem;
     margin: 0.5rem 0;
+    overflow: scroll;
```

Comment:
some sc values are really long, so allowing this div to scroll to allow users to see the value